tools/lint/lint.py

```diff
@@ -134,6 +134,21 @@ def check_ahem_copy(repo_root, path):
     return []
 
 
+def check_git_ignore(repo_root, path):
```

Comment:
Is there a way to make this work calling multiple paths at once? This seems like it isn't going to help performance.